linera-sdk/src/ethereum.rs

```diff
@@ -14,25 +14,27 @@ pub use linera_ethereum::{
 use linera_ethereum::{client::JsonRpcClient, common::EthereumServiceError};
 use serde::{Deserialize, Serialize};
 
-use crate::contract::wit::contract_system_api;
+use crate::{contract::wit::contract_system_api, service::wit::service_system_api};
+
+// TODO(#3143): Unify the two types into a single `EthereumClient` type.
```

Comment:
Does the `Contract` one still make sense at all? With the planned change for the HTTP oracle, I imagine it wouldn't work anymore?